scripts/establish_conda_env.sh

```diff
@@ -41,13 +41,38 @@ function establish_OS ()
 	esac
 }
 
+function read_ravenrc ()
```

Comment:
This changes is available in another PR, right?

Reply:
Yes, it came up while working on this stuff. Since others are depending on that feature in this branch, I recommend reviewing it here.